server/app/urls.go

```diff
@@ -0,0 +1,43 @@
+package app
+
+import "fmt"
+
+const pluginRoute = "plug"
+
+func getRunsURL(siteURL string, manifestID string) string {
+	return fmt.Sprintf("%s/%s/%s/runs",
+		siteURL,
+		pluginRoute,
+		manifestID,
+	)
```

Comment:
nit: Might we consider formatting as:
```suggestion
	return fmt.Sprintf(
              "%s/%s/%s/runs",
		siteURL,
		pluginRoute,
		manifestID,
	)
```